GATEWAY/src/providers/tilesApi.service.ts

```diff
@@ -183,21 +183,37 @@ export class TilesApi {
    * @param {string} virtualTileId - The virtual tile
    * @param {string} applicationId - The application the virtual tile is registered to
    */
-  public pairDeviceToVirualTile = (deviceId: string, virtualTileId: string): Promise<Response> => {
+  public pairDeviceToVirualTile = (deviceId: string, virtualTileId: string): Promise<void> => {
```

Comment:
it used to return a response, but now it doesn't anymore. I can make it return the result, but we do not need it for anything else